manim/__main__.py

```diff
@@ -14,7 +14,8 @@
 
 try:
     from manim.grpc.impl import frame_server_impl
-except ImportError:
+except ImportError as e:
+    print(e)
```

Comment:
don't print it.

Reply:
use something like logging and set it to debug mode or just use `pass` here.

Reply:
I changed it to info. People aren't likely to try printing in debug mode if manim doesn't start.

Reply:
That's optional dependency right? If yes, then it should be debug, if no info is ok.

Reply:
Really it just doesn't make much sense to have this here. I moved it to the place before it's used so it'll only be printed if the flag is passed.